### PR TITLE
Bump prismatic/schema version

### PR DESCRIPTION
**Pull request by phillipm**
To allow compatibility with `clojure 1.9.0 beta1`

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,11 +1,11 @@
-(defproject prismatic/schema-generators "0.1.2-SNAPSHOT"
+(defproject prismatic/schema-generators "0.1.3-SNAPSHOT"
```
Comment (w01fe) [inline project.clj]: This was correct as-is; 0.1.2 will be the next release.

Comment (phillipm) [reply project.clj]: updated

**Comment by phillipm** [comment]
@w01fe when you get the chance, can you publish this to maven? Thanks!

**Comment by w01fe** [comment]
I will when I get a chance; but for my understanding, why do you need a release?  (I think you can always choose to explicitly depend on a newer version of schema and override the transitive dependency.).  Thanks

**Comment by phillipm** [comment]
Ah, you're right, overriding the transitive dependency should work well; forgot that was an option :)

**Comment by w01fe** [comment]
Great, thanks.  I might wait to cut one until there's an official Clojure 1.9 release then, in case there are any further incompatibilities.

**Pull request merged**